### DISCUSSION TITLE
Login does not change language

### DISCUSSION THREAD

**Discussion by fsan2val** [AzuraCast/AzuraCast#7047] [General]
Hello

When updating to the latest version I changed the language of the login buttons, even though I have it in Spanish it does not change those two words, see image

Thanks
![Inicia-sesión](https://github.com/AzuraCast/AzuraCast/assets/69484466/9ba4e6f7-2c69-46ba-bf4c-ed4b4d529594)

**Comment by BusterNeece** [comment]
Confirmed fixed in the latest Rolling Release.
